index.html

```diff
@@ -64,8 +245,29 @@ <h1>I’m Lilian<br>
           </li>
         </ul>
       </div>
+      <div id="iteme-7"><button class="buttonresume">Get my Resume</button></div>
+      <div class="my-accordion">
+        <ul>
+          <li class='menu'>Languages
+
+
```

Comment:
- [ ] OPTIONAL 
You left an extra empty line in your code that is serving no purpose. To ensure a cleaner code let's remove the extra empty lines found throughout the code.